docs/user/ml/index.asciidoc

```diff
@@ -223,3 +223,8 @@ point type selector to filter the results by specific types of change points.
 
 [role="screenshot"]
 image::user/ml/images/ml-change-point-detection-selected.png[Selected change points]
+
+You can attach selected charts to a dashboard or a case by using the 
+context menu. You can set the applied time range and the number of change points
```

Comment:
I reckon it's worth mentioning that you can configure specific partitions _**or**_ set a number of top change points to plot

Reply:
Updated via [a108621](https://github.com/elastic/kibana/pull/164277/commits/a108621a3cd5e6cdb68c59f4124e723372e3563a)